### PR TITLE
E2E test cases for waypoint management

### PR DESCRIPTION
**Pull request by YaoZengzeng**
**What type of PR is this?**

/kind enhancement

**What this PR does / why we need it**

e2e tests for waypoint management:

1. Add/remove waypoints at service granularity
2. Add/remove waypoints at namespace granularity
3. Add/remove waypoints at pod granularity

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: test/e2e/baseline_test.go [hunk 1]
```diff
@@ -443,6 +448,61 @@ spec:
...
+
+		t.NewSubTest("before").Run(func(t framework.TestContext) {
+			dst := apps.EnrolledToKmesh
+			for _, src := range apps.All {
+				if src.Config().IsUncaptured() {
+					continue
+				}
+				t.NewSubTestf("from %v", src.Config().Service).Run(func(t framework.TestContext) {
+					c := IsL4()
+					if src.Config().HasSidecar() {
```
Comment (hzxuzhonghu) [inline test/e2e/baseline_test.go:467]: nit: remove, always check L4 here

**Review by hzxuzhonghu — DISMISSED** [review]

File: test/e2e/baseline_test.go [hunk 2]
```diff
@@ -443,6 +448,132 @@ spec:
...
+				}
+
+			}
+		})
+	})
+}
+
+// Test add/remove waypoint at ns or service granularity.
+func TestRemoveAddNsOrServiceWaypoint(t *testing.T) {
+	for _, granularity := range []Granularity{Service /*,Namespace*/} {
```
Comment (YaoZengzeng) [inline test/e2e/baseline_test.go:516]: @hzxuzhonghu maybe this PR could be merged first, when https://github.com/kmesh-net/kmesh/issues/628 get fixed, then we uncomment `Namespace`

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**